mirrord/cli/src/container.rs

```diff
@@ -193,6 +192,41 @@ async fn create_sidecar_intproxy(
             )
         })?;
 
+    if matches!(base_command.runtime(), ContainerRuntime::Docker) {
```

Comment:
Could you add some docs here, saying that we're checking the sidecar because sometimes it doesn't start properly (was this the issue?)